website/docs/getting-started/setup/installation-guides/aws-ami.md

```diff
@@ -1,145 +1,113 @@
 ---
-description: Deploy Appsmith on AWS with an AMI on the marketplace
+description: Deploy Appsmith on AWS with an AMI available on the AWS marketplace.
 sidebar_position: 4
 ---
 
 # AWS AMI
+This page provides steps to install Appsmith using an Amazon Machine Image (AMI).
```

Comment:
should we link to the official website?

Reply:
The links are available in the Install Appsmith section.